sdk/notificationhubs/notification-hubs/test/public/createOrUpdateInstallation.spec.ts

```diff
@@ -1,42 +1,43 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { describe, it, assert, beforeEach } from "vitest";
 import {
   NotificationHubsClientContext,
   createOrUpdateInstallation,
   deleteInstallation,
-} from "@azure/notification-hubs/api";
-import { assert, isNode } from "@azure/test-utils";
+} from "../../src/api/index.js";
 import { Recorder } from "@azure-tools/test-recorder";
-import { createAppleInstallation } from "@azure/notification-hubs/models";
+import { createAppleInstallation } from "../../src/models/index.js";
 import { createRecordedClientContext } from "./utils/recordedClient.js";
+import { isNode } from "@azure/core-util";
 
 describe("createOrUpdateInstallation()", () => {
   let recorder: Recorder;
   let context: NotificationHubsClientContext;
   const installationId = "0e7c5973-714c-4ba9-a233-7c4497d5f43b";
   const pushChannel = "00fc13adff785122b4ad28809a3420982341241421348097878e577c991de8f0";
 
-  beforeEach(async function (this: Mocha.Context) {
+  beforeEach(async (_ctx) => {
```

Comment:
remove the context variable?

Reply:
This is broken and won't work until I can pass in the context. So, basically that's a TODO once test-recorder supports vitest.